onBoardProject/screens/login_screen.tsx

```diff
@@ -0,0 +1,87 @@
+import React, {Component, useState} from 'react';
+import {
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+  StyleSheet,
+  Text,
+  TextInput,
+  useColorScheme,
+  View,
+  Pressable,
+} from 'react-native';
+
+type MyProps = {
+  title: string;
+  email: string;
+  password: string;
+};
+
+export const LoginScreen: React.FC<{
+  title: string;
+}> = ({title}) => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  return (
+    <View style={styles.loginView}>
+      <Text style={styles.titleStyle}> {title} </Text>
```

Comment:
```suggestion
      <Text style={styles.titleStyle}>{title}</Text>
```

Esses espaços q vc deu o componente `Text` são parte do texto. Tudo o que fica entre as tags (no children) é uma string que ele lê. Por isso ficou à direita o texto: a string resultante ficou `" Bem vindo(a) à Taqtile! "`